test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/RunnableProjectConfigTests.cs

```diff
@@ -143,5 +165,120 @@ public async Task PerformTemplateValidation_ChoiceValuesValidation(string paramD
                     validationError.ErrorMessage);
             }
         }
+
+        [Fact]
+        public void VerifyComputedSymbolsParsedCorrectly()
+        {
+            //
+            // Template content preparation
+            //
+
+            //fill test data here
+            TemplateConfigModel config = new TemplateConfigModel("test")
+            {
+                Name = "name",
+                ShortNameList = new[] { "shortName" },
+                Symbols = new[]
+                {
+                    new ComputedSymbol("computed1", JObject.Parse(ValidComputedDefinition)),
+                    new ComputedSymbol("computed2", JObject.Parse(ValidComputedDefinition))
+                }
+            };
+
+            //
+            // Dependencies preparation and mounting
+            //
+
+            IEngineEnvironmentSettings environmentSettings = _environmentSettingsHelper.CreateEnvironment();
+
+            string sourceBasePath = environmentSettings.GetTempVirtualizedPath();
+            using IMountPoint mountPoint = environmentSettings.MountPath(sourceBasePath);
+            using RunnableProjectConfig templateConfig = new RunnableProjectConfig(environmentSettings, new RunnableProjectGenerator(), config, mountPoint.Root);
+
+            //verify
+            Assert.Equal(7, templateConfig.GlobalOperationConfig.Macros.Count);
+            Assert.Equal(2, templateConfig.GlobalOperationConfig.Macros.Count(m => m.Type == "evaluate"));
+            // default symbols are different for OS
+            templateConfig.GlobalOperationConfig.SymbolNames.Count.Should().BeGreaterThanOrEqualTo(3);
```

Comment:
it would be good to check if at least computed symbols are there

Reply:
same applies to tests below

Reply:
done